modules/terminations.stanza

```diff
@@ -28,6 +28,43 @@ public pcb-enum ocdb/terminations/LVDSTerminationMethod :
   LVDSTerminationACDualResistorSplitCap
   LVDSTerminationACDualResistorSplitReduced
 
+public pcb-enum ocdb/terminations/USB-C-ConfigurationChannel :
+  SourceDefault
+  Source5V-1A5
+  Source5V3A0
+  SinkDefault
+
+; Terminates USB-C based on pull voltage and USB-C configuration
+public defn terminate-usb-c (usb-c-bundle:JITXObject, pull-up-net:JITXObject, pull-up-voltage:Double, method:USB-C-ConfigurationChannel) :
```

Comment:
We should move away from using Doubles as voltages and replace them with Toleranced. So this might convert to:
```
pull-up-voltage:Toleranced
```
and the invocation could be something like `typ(5.0v)`

Reply:
fixed